src/download.test.ts

```diff
@@ -114,9 +120,15 @@ describe("getSkylinkUrl", () => {
   });
 
   it("should return correctly formed URLs with forced download and path", () => {
-    expect(client.getSkylinkUrl(skylink, { download: true, path: "foo?bar" })).toEqual(
-      `${expectedUrl}/foo%3Fbar${attachment}`
-    );
+    const url = client.getSkylinkUrl(skylink, { download: true, path: "foo?bar" });
+
+    expect(url).toEqual(`${expectedUrl}/foo%3Fbar${attachment}`);
+  });
+
+  it("should return correctly formed URLs with no-response-metadata set", () => {
+    const url = client.getSkylinkUrl(skylink, { noResponseMetadata: true });
+
+    expect(url).toEqual(`${expectedUrl}?no-response-metadata=true`);
```

Comment:
very nitty nit: For composition cases like this one `toContain` might be better than `toEqual` but I don't think we need to change it right now. It might be helpful in case where we add default query params or something similar.

Reply:
I would argue that `toEqual` is generally better as it catches any changes to the value. Then we have the chance to confirm that the change was desired and update the test if so. You make a good point about default query params, we can address that when/if we get to it.

Reply:
Also I think `toContain` wouldn't quite work unless no-response-metadata (in this example) was always first. We'd have to come up with something else, maybe even a custom matcher.

Reply:
`.toContain("no-response-metadata=true")` :D

I agree with the rest of your point. Definitely no need to change anything now.